library/spark/spark-accumulo-library/src/main/java/uk/gov/gchq/gaffer/sparkaccumulo/operation/handler/dataframe/GetDataFrameOfElementsHandler.java

```diff
@@ -44,6 +45,14 @@
         } else {
             operationOptions = new HashMap<>();
         }
+
+        for (final SchemaEntityDefinition entityDef : store.getSchema().getEntities().values()) {
```

Comment:
This could be annoying for people. Could you use the keyword VERTEX instead? Do you also need to check GROUP, SOURCE, DESTINATION and DIRECTED too?

Reply:
This issue builds on the implementation of the GetDataFrameOfElements operation which relies on these keywords to create Rows (for the DataFrame) from the Elements in the Gaffer graph. Any changes applied here would also need to be made to the GetDataFrameOfElements operation.